device-types/Cisco/ME-3600X-24CX-M.txt

```diff
@@ -0,0 +1,75 @@
+---
+manufacturer: Cisco
+model: ME-3600X-24CX-M
+slug: me-3600x-24cx-m
+part_number: ME-3600X-24CX-M
+is_full_depth: false
+u_height: 1
+console-ports:
+  - name: Console
+    type: rj-45
+interfaces:
+  - name: Gi0
+    type: 1000base-t
+    mgmt_only: true
+  - name: Gi0/1
+    type: 1000base-x-sfp
+  - name: Gi0/2
+    type: 1000base-x-sfp
+  - name: Gi0/3
+    type: 1000base-x-sfp
+  - name: Gi0/4
+    type: 1000base-x-sfp
+  - name: Gi0/5
+    type: 1000base-x-sfp
+  - name: Gi0/6
+    type: 1000base-x-sfp
+  - name: Gi0/7
+    type: 1000base-x-sfp
+  - name: Gi0/8
+    type: 1000base-x-sfp
+  - name: Gi0/9
+    type: 1000base-x-sfp
+  - name: Gi0/10
+    type: 1000base-x-sfp
+  - name: Gi0/11
+    type: 1000base-x-sfp
+  - name: Gi0/12
+    type: 1000base-x-sfp
+  - name: Gi0/13
+    type: 1000base-x-sfp
+  - name: Gi0/14
+    type: 1000base-x-sfp
+  - name: Gi0/15
+    type: 1000base-x-sfp
+  - name: Gi0/16
+    type: 1000base-x-sfp
+  - name: Gi0/17
+    type: 1000base-x-sfp
+  - name: Gi0/18
+    type: 1000base-x-sfp
+  - name: Gi0/19
+    type: 1000base-x-sfp
+  - name: Gi0/20
+    type: 1000base-x-sfp
+  - name: Gi0/21
+    type: 1000base-x-sfp
+  - name: Gi0/22
+    type: 1000base-x-sfp
+  - name: Gi0/23
+    type: 1000base-x-sfp
+  - name: Gi0/24
+    type: 1000base-x-sfp
+  - name: Te0/1
+    type: 10gbase-x-sfpp
+  - name: Te0/2
+    type: 10gbase-x-sfpp
+  - name: Te0/3
+    type: 10gbase-x-sfpp
+  - name: Te0/4
+    type: 10gbase-x-sfpp
+power-ports:
+  - name: PSU1
+    type: PWR-ME3KX-AC
+  - name: PSU2
+    type: PWR-ME3KX-AC
```

Comment:
* ME PSU's are hot swappable and should be modelled as modules
* These are not a valid type